cli/pom.xml

```diff
@@ -141,6 +145,21 @@
     </testResources>
     <plugins>
       <plugin>
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+            <configuration>
+              <classifier>minimal</classifier>
+            </configuration>
```

Comment:
@jdcasey : Due to the standard jar embedded logging dependencies I needed a standard jar for use in the integration tests. Naming might not be ideal....